compatibility_spark-3/src/test/scala/za/co/absa/hyperdrive/compatibility/impl/writer/cdc/hudi/TestHudiCDCToSCD2Writer.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.compatibility.impl.writer.cdc.hudi
+
+import org.apache.spark.sql.streaming.Trigger
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest.{FlatSpec, Matchers}
+import za.co.absa.hyperdrive.compatibility.impl.writer.cdc.hudi.scd2.HudiCDCToSCD2Writer
```

Comment:
Looks like this test is in the wrong package. Please move it into the corresponding package like the tested class.

Reply:
Done